pinot-controller/src/test/java/com/linkedin/pinot/controller/api/restlet/resources/SegmentCompletionProtocolDeserTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.controller.api.restlet.resources;
+
+import com.linkedin.pinot.common.protocols.SegmentCompletionProtocol;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class SegmentCompletionProtocolDeserTest {
```

Comment:
Good job adding this test. You also need to add a test on response serialization and deserialization,.